README.md

```diff
@@ -38,6 +40,7 @@ ggplot(cluster_jobchange, aes(x = reorder(name, jobchange), y = jobchange, fill
   geom_col() +
   coord_flip() +
   theme_cmap()
+#> 'Whitney' font family not found. Using a substitute...
```

Comment:
Just want to point this out. We should try to avoid re-knitting on machines without Whitney. Not a big deal right now, especially since none of us currently have access to Whitney, but just something to keep in mind.